store/mockstore/mocktikv/rpc.go

```diff
@@ -245,14 +245,30 @@ func (h *rpcHandler) handleKvGet(req *kvrpcpb.GetRequest) *kvrpcpb.GetResponse {
 }
 
 func (h *rpcHandler) handleKvScan(req *kvrpcpb.ScanRequest) *kvrpcpb.ScanResponse {
-	if !h.checkKeyInRegion(req.GetStartKey()) {
-		panic("KvScan: startKey not in region")
-	}
 	endKey := h.endKey
```

Comment:
Do we need to set `endKey` to `h.startKey` when `Reverse` is true?

Reply:
No, `endKey := h.endKey` here means the region end key, It has nothing to do with `reverse` scan.

Reply:
Got it.